packages/shared/src/wallet/components.tsx

```diff
@@ -0,0 +1,71 @@
+import type { BigNumber } from 'bignumber.js'
+import { FC, Fragment } from 'react'
+import { formatBalance, formatChecksumAddress, formatCurrency, formatEthereumAddress, formatToken } from './formatter'
+
+interface FormattedBalanceProps {
```

Comment:
Those interface should also be exported otherwise use site cannot name those props.